consensus/src/msg_handler.rs

```diff
@@ -8,30 +8,25 @@ use crate::commons::{ConsensusError, RoundUpdate};
 use crate::iteration_ctx::RoundCommittees;
 use crate::user::committee::Committee;
 use async_trait::async_trait;
-use node_data::ledger::to_str;
-use node_data::message::{Message, MessageTrait, Status};
+use node_data::message::{Message, Status};
 use node_data::StepName;
-use std::fmt::Debug;
 use tracing::{debug, trace};
 
 /// Indicates whether an output value is available for current step execution
-/// (Step is Ready) or needs to collect data (Step is Pending)
-pub enum HandleMsgOutput {
```

Comment:
I disagree with this change.  I chose Ready/Pending to reflect a pattern from Future task which is more descriptive to the  step execution. https://doc.rust-lang.org/std/task/enum.Poll.html

Reply:
Ok, but I see that the inner message of the Pending enum is never used.

Do you agree if I rollback and remove that inner message?

Reply:
Of course, if the inner of Pending is unused, we should remove it. In that way, it duplicates the Poll enum/pattern even more.